data/json/items/tool/lighting.json

```diff
@@ -624,41 +611,37 @@
     "type": "TOOL",
     "name": { "str": "heavy-duty flashlight (off)", "str_pl": "heavy-duty flashlights (off)" },
     "description": "A heavy-duty aluminum LED flashlight, often used by security guards.  It even works underwater, and it's not bad for smacking hooligans.",
-    "weight": "450 g",
-    "volume": "470 ml",
+    "weight": "720 g",
+    "volume": "5468 ml",
```

Comment:
Typo in volume value

Reply:
Did you use `Package Dimensions` instead of `Item Dimensions` ?

Also, the thing is not exactly rectangular.
`Item Dimensions LxWxH	‎9.84 x 4.92 x 6.89 inches`
If we approximate it as oval cylinder of those dimensions, volume will be `4.92/2*6.89/2*math.pi*9.84*2.5*2.5*2.5` = `4093` ml

If we approximate it as ‎9.84 x 4.92 x 4.92 cylinder:
2923 ml for the cylinder. 
And if we eyeball handle to be 3/4 long, 6.89-4.92 tall and 4.92/2 wide rectangle:
`(6.89-4.92)*(9.84*3/4)*(4.92/2)*2.5*2.5*2.5` = 559 ml

3482 ml total
Still feels off. Must be having a lot of air inside.

Reply:
i suspect the biggest volume is from the handle

Reply:
Recalculated using 1 inch=2.54cm. I can settle on 3652 or 4293 or something in between those 2 values.
The thing is certainly not a rectangular prism